### DISCUSSION TITLE
Conditional action is failing even though the syntax is correct

### DISCUSSION THREAD

**Discussion by gautamkrishnar** [community/community#26052] [Actions]
<p>A repo has a yml file as follows:</p>
<pre><code class="lang-auto">name: Notify the Gchat Group for PRs from main repo
on:
  pull_request:
    types: [opened]
jobs:
  notify:
    runs-on: ubuntu-latest
    steps:
      - name: Google Chat Notification
        if: github.repository == ‘1-Platform/op-components’
        uses: gautamkrishnar/google-chat-notification@master
        with:
          url: ${{ secrets.GOOGLE_CHAT_WEBHOOK }}

</code></pre>
<p>But this action is failing with <code>The workflow is not valid. .github/workflows/notify-main.yml (Line: 10, Col: 13): Unexpected symbol: '‘1-Platform/op-components’'. Located at position 22 within expression: github.repository == ‘1-Platform/op-components’</code> error. Is this a bug?</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/1-Platform/op-components/actions/runs/223647381" target="_blank" rel="nofollow noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/8397274/181094992-6e2f19bc-540b-4bb7-ae78-11a59d5157b2.png" class="thumbnail onebox-avatar" width="315" height="315">

<h3><a href="https://github.com/1-Platform/op-components/actions/runs/223647381" target="_blank" rel="nofollow noopener">1-Platform/op-components</a></h3>

<p>One platform component library. Contribute to 1-Platform/op-components development by creating an account on GitHub.</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/gautamkrishnar">@gautamkrishnar</a>,</p>
<p>This is due to the incorrect single quote in the if expression.<br>
Please change to use:</p>
<pre><code class="lang-auto">if: github.repository == '1-Platform/op-components'   # fix the single quote
</code></pre>
<p>Thanks.</p>

**Comment by gautamkrishnar** [comment]
<p>Thanks for the solution</p>
